code/modules/mob/living/living.dm

```diff
@@ -10,16 +10,29 @@
 
 //mob verbs are faster than object verbs. See above.
 /mob/living/pointed(atom/A as mob|obj|turf in view())
+	if(!isturf(src.loc) || !(A in range(world.view, get_turf(src))))
+		return FALSE
 	if(src.stat || !src.canmove || src.restrained())
-		return 0
+		return FALSE
 	if(src.status_flags & FAKEDEATH)
-		return 0
-	if(!..())
-		return 0
-
-	src.visible_message("<b>[src]</b> points to [A].")
-	return 1
+		return FALSE
+
+	face_atom(A)
+	if(isturf(A))
+		if(pointing_effect)
+			clear_point()
+		pointing_effect = new /obj/effect/decal/point(A)
+		pointing_effect.invisibility = invisibility
+		addtimer(CALLBACK(src, .proc/clear_point), 20)
+	else
+		var/pointglow = filter(type = "drop_shadow", x = 0, y = -1, offset = 2, size = 1, color = "#F00")
+		LAZYADD(A.filters, pointglow)
+		addtimer(CALLBACK(src, .proc/remove_filter, A, pointglow), 20)
+	visible_message("<b>\The [src]</b> points to \the [A].")
+	return TRUE
```

Comment:
Why not also add the pointing effect here? I figure the drop shadow itself isn't obvious enough tbh.

Reply:
doing both adds a ton of ugly visual clutter. i believe the drop shadow is bright enough to be recognizable